crates/script/src/verify.rs

```diff
@@ -237,8 +238,8 @@ async fn verify_contracts(
         let num_verifications = future_verifications.len();
         let mut num_of_successful_verifications = 0;
         sh_println!("##\nStart verification for ({num_verifications}) contracts")?;
-        for verification in future_verifications {
-            match verification.await {
+        for verification in join_all(future_verifications).await {
+            match verification {
```

Comment:
I think this is currently done as a naive back-off to make sure we don't spam the verify endpoint considering that is the most common failure.

Reply:
yeah but this could spam an already fragile endpoint, i don't think we should do this